scripts/chk_shellscripts/ignore.txt

```diff
@@ -1,39 +1,21 @@
-./test/docsCodeStyle.sh
 ./test/cmdlineTests.sh
-./scripts/isoltest.sh
-./scripts/get_version.sh
 ./scripts/soltest.sh
-./scripts/test_emscripten.sh
 ./scripts/wasm-rebuild/docker-scripts/rebuild_tags.sh
 ./scripts/wasm-rebuild/docker-scripts/rebuild_current.sh
 ./scripts/wasm-rebuild/docker-scripts/genbytecode.sh
-./scripts/wasm-rebuild/docker-scripts/patch.sh
-./scripts/wasm-rebuild/rebuild.sh
-./scripts/build_emscripten.sh
 ./scripts/ci/build_emscripten.sh
-./scripts/docker_build.sh
 ./scripts/docs_version_pragma_check.sh
 ./scripts/uniqueErrors.sh
 ./scripts/tests.sh
-./scripts/docker_deploy.sh
 ./scripts/bytecodecompare/storebytecode.sh
 ./scripts/deps-ppa/static_z3.sh
 ./scripts/ASTImportTest.sh
 ./scripts/install_static_z3.sh
-./scripts/install_obsolete_jsoncpp_1_7_4.sh
 ./scripts/install_deps.sh
-./scripts/build.sh
-./scripts/run_proofs.sh
 ./scripts/common_cmdline.sh
 ./scripts/docker_deploy_manual.sh
 ./scripts/endToEndExtraction/create_traces.sh
-./scripts/release.sh
-./scripts/download_ossfuzz_corpus.sh
 ./scripts/release_ppa.sh
-./scripts/install_cmake.sh
-./scripts/release_emscripten.sh
 ./scripts/create_source_tarball.sh
-./scripts/docs.sh
 ./.circleci/soltest.sh
-./.circleci/osx_install_dependencies.sh
 ./.circleci/soltest_all.sh
```

Comment:
Just curious if a PR is planned to set the remaining scripts in ignore right as well? If so, I'd prefer reviewing the bunch in one go :-)

Reply:
Yeah. I have a local branch that fixes all the remaining issues and removes the ignore list but I think it's hard to review as a whole because there are too many unrelated changes so I wanted to split it into smaller PRs. The downside is of course that shellcheck won't be passing checking everything until the very last one is merged.

I especially wanted to deal with quoting first because it adds too much noise and makes other small changes easy to miss. And it's important that they get reviewed carefully because some of these scripts are not hooked into CI.

In this PR I think you should focus mostly on whether changes that are present are correct. The ones that are missing will be caught by the final shellcheck run in the last PR.

If you want to see that last run after all the changes, I can push my remaining two PRs and put them on top of each other so that you can see that it's clean in the topmost one. Originally I didn't want to do this to keep #10586 directly on develop so that it can be reviewed in parallel but well, it's blocked anyway.

Reply:
Let's do it as you suggest since hurrying a review of the bunch may not be wise. So, let's review one by one and wait for the last PR to pass.

Reply:
I have pushed all the branches and the last one removes the whole ignore list:
- #10585 - this PR
- #10877 - fixes the remaining warnings about quotes by using Bash arrays
- #10586 - huge pile of miscellaneous fixes for shellcheck warnings
- #10879 - final PR that disables the few remaining warnings locally in each script and removes the ignore list

I will keep them on top of each other so that you can always check `chk_coding_style` in #10879 to make sure that all warnings get fixed eventually.